Bjorncs/jdisc access log

bjorncs opened the pull request.

bjorncs committed:
Add unit test verifying that the access log is invoked for 414

bjorncs committed:
Retrieve status code and content length using low-level Jetty interface

Response.getStatus/getReturnedContentSize is not populated for requests
rejected by Jetty.

gjoranv commented:
It seems that you mixed the commit messages for the first and third commit. Please fix using git rebase and 'edit'.

bjorncs committed:
Register access log directly on Server

The access log must be registered on the Server instance to be able to
log requests that are rejected by Jetty (e.g 414).

bjorncs commented:
I have fixed the commit messages. PTAL
